### PR TITLE
Lab 04

### PR DESCRIPTION
**Pull request by sak797**
We discussed alternative view of Matrix multiplication i.e alignment between each row and the input vector. Then we discussed how we deal with longer input which is normally found in natural signals. To do matrix multiplication on longer input we converted each row of the original matrix to Toeplitz matrix. Then we had a demonstration where we were finding the tones of audio file using convolution.

Lab 04 2/20/19
Mimee Xu , Sai Anirudh Kondaveeti, Rui Jiang(rj1407)

### CODE REVIEW

**Comment by Atcold** [comment]
Why are there 3 PRs for Lab04?
There's no description or authors list. Plz, fix trying to check how others have been competing this assignment.

**Comment by sak797** [comment]
> Why are there 3 PRs for Lab04?

I wasn't sure how to update the original PR. I followed the piazza post to make my changes to the latest copy.

> There's no description or authors list. Plz, fix trying to check how others have been competing this assignment.

I have updated the description and authors.

File: main.tex [hunk 1]
```diff
@@ -1,12 +1,10 @@
 \documentclass[oneside]{book}
+\usepackage{float}
 \input{preamble}
 \input{jupyter}
 
 \title{SP19 DL collaborative notes}
 \author{
-  The students of SP19 Deep Learning\\
```
Comment (Atcold) [inline main.tex]: Why was this removed???

File: main.tex [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
-  Editor: Alfredo Canziani\\
```
Comment (Atcold) [inline main.tex]: I'm editing this stuff… 😑

File: main.tex [hunk 3]
```diff
@@ -1,12 +1,10 @@
...
-  NYU
```
Comment (Atcold) [inline main.tex]: And we're at NYU.

Comment (sak797) [reply main.tex]: I didn't realize this was removed. Sorry for that. Will make the change

File: main.tex [hunk 4]
```diff
@@ -1,4 +1,5 @@
 \documentclass[oneside]{book}
+\usepackage{float}
```
Comment (Atcold) [inline main.tex:2]: I thought this package was already used by other editors. Hmm...

**Comment by Atcold** [comment]
Thanks.
